src/Serilog.Formatting.Elasticsearch/ExceptionAsObjectJsonFormatter.cs

```diff
@@ -42,15 +42,18 @@ public class ExceptionAsObjectJsonFormatter : ElasticsearchJsonFormatter
         /// <param name="formatProvider">Supplies culture-specific formatting information, or null.</param>
         /// <param name="serializer">Inject a serializer to force objects to be serialized over being ToString()</param>
         /// <param name="inlineFields">When set to true values will be written at the root of the json document</param>
+        /// <param name="renderMessageTemplate">If true, the message template will be rendered and written to the output as a
+        /// property named RenderedMessageTemplate.</param>
         /// <param name="formatStackTraceAsArray">If true, splits the StackTrace by new line and writes it as a an array of strings</param>
         public ExceptionAsObjectJsonFormatter(bool omitEnclosingObject = false, 
             string closingDelimiter = null, 
             bool renderMessage = false, 
             IFormatProvider formatProvider = null, 
             ISerializer serializer = null, 
             bool inlineFields = false, 
-            bool formatStackTraceAsArray = false) 
-            : base(omitEnclosingObject, closingDelimiter, renderMessage, formatProvider, serializer, inlineFields, true, formatStackTraceAsArray)
+            bool formatStackTraceAsArray = false,
+            bool renderMessageTemplate = true)
```

Comment:
This is a breaking change; introducing a new (even optional) parameter. Why do you want to change this?